### DISCUSSION TITLE
Give permission to unauthorized users to view wiki content

### DISCUSSION THREAD

**Discussion by TSLabPRO** [requarks/wiki#2787] [Help / Questions] [answered]
Hi,

I need to give permission to unauthorized users to view wiki content. Found that earlier this setting could be enabled in the config file (set the public: true in config.yml). Now there is no such item in this file. I also did not find such a setting in administration.
Thanks.

**Host Info (please complete the following information):**

    OS: Docker Container (Linux)
    Wiki.js version: 2.5.170
    Database engine: PostgreSQL 11.10

**Comment by chris-hs** [comment, marked as answer]
If you want to allow unauthorized aka guest users to view all content, you have to edit **Permissions** and **Page Rules** for the **Guest** user group. I would start with the following settings and see if your guest users can access what they need to.

Allowed **Permissions**
- read:pages
- read:assets
- read:comments
- write:comments

**Page Rules**
Add rule for *Path Starts With...* `/` and add the following roles (permissions).
- read:pages
- read:assets
- read:comments
- write:comments

**Comment by LetArch** [comment]
I turn like this permissions for group "Guests" and users from this group don't see assets of pages. What's do it?
